backend/utils.ts

```diff
@@ -342,6 +343,25 @@ export const stringWithAbbreviationsProcessor = (string: string): string => {
   return establishmentName
 }
 
+// Either explicitly enable redirect for enabled prisons or check for the developer role
+export const isRedirectEnabled = (res): boolean => {
+  const redirectUrl = config.app.prisonerProfileRedirect.url
+  const redirectDate = config.app.prisonerProfileRedirect.enabledDate
+  return (
+    redirectUrl &&
+    ((redirectDate && redirectDate < Date.now()) ||
+      res.locals.user?.userRoles?.some((role) => role === 'ROLE_DPS_APPLICATION_DEVELOPER'))
+  )
+}
+
+export const isRedirectCaseLoad = (activeCaseLoadId: string): boolean => {
+  return config.app.prisonerProfileRedirect.enabledPrisons?.split(',')?.includes(activeCaseLoadId)
+}
+
+export const useNewProfile = (redirectCaseLoadEnabled: boolean, redirectEnabled: boolean): boolean => {
+  return !!(redirectCaseLoadEnabled && redirectEnabled)
+}
```

Comment:
These arguments were being used in the wrong order which was confusing when looking into why the tests weren't passing
```suggestion
export const useNewProfile = (redirectEnabled: boolean, redirectCaseLoadEnabled: boolean): boolean =>
  redirectCaseLoadEnabled && redirectEnabled
```